.github/workflows/treetracker-query-api-build-deploy-dev.yml

```diff
@@ -20,10 +20,10 @@ jobs:
       - uses: actions/checkout@v2
         with:
           ref: ${{ github.event.inputs.git-tag }}
-      - name: Use Node.js 16.x
+      - name: Use Node.js 18.x #semantic-release
         uses: actions/setup-node@v1
         with:
-          node-version: '16.x'
+          node-version: '18.x'
```

Comment:
If we need to upgrade the node version, I suggest we put it into a dedicated PR to do it, the smaller and more focused PR, the better

Reply:
The PR wasn't for that. My virtual machine that must have upgraded to work.